pom.xml

```diff
@@ -216,6 +217,62 @@
     </build>
 
     <profiles>
+		<profile>
+			<id>rpm</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>rpm-maven-plugin</artifactId>
+						<version>2.1.5</version>
+						<executions>
+							<execution>
+								<id>generate-rpm</id>
+								<goals>
+									<goal>rpm</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<group>Applications/Internet</group>
+							<packager>Confluent Packaging</packager>
+							<needarch>false</needarch>
+							<requires>
+								<require>confluent-common</require>
+							</requires>
+							<mappings>
+								<mapping>
+									<directory>/usr/share/java/</directory>
```

Comment:
good catch, I actually had this fixed in a previous commit and forgot to pull it down when I moved the plugin to it's own profile. 

the paths should have the suffix ${project.artifactId}

after putting it back we can see that only the proper paths are included with rpm -qpl